server/api/model/tree_permit.js

```diff
@@ -0,0 +1,41 @@
+const Model = require('./base_model');
+const tpc = require('./tree_permit_constants');
+class TreePermit extends Model {
+
+	get rules() {
+		return {
+
+			[tpc.REGIONAL_OFFICE]: 'string',
```

Comment:
all property types are string?

Reply:
yes, as we don't want to suppose any assumptions on the input. if we had coordinates, it would have been of course modelled accordingly